variables.sh

```diff
@@ -6,17 +6,18 @@ NAMESPACE=${NAMESPACE:-namely}
 CONTAINER=${DOCKER_REPO}${NAMESPACE}
 LATEST=${1:false}
 
-GO_VERSION=${GO_VERSION:-1.14}
-GRPC_VERSION=${GRPC_VERSION:-1.39}
-GRPC_JAVA_VERSION=${GRPC_JAVA_VERSION:-1.39}
-GRPC_WEB_VERSION=${GRPC_WEB_VERSION:-1.2.1}
-GRPC_GATEWAY_VERSION=${GRPC_GATEWAY_VERSION:-2.0.1}
-UBER_PROTOTOOL_VERSION=${UBER_PROTOTOOL_VERSION:-1.3.0}
-SCALA_PB_VERSION=${SCALA_PB_VERSION:-0.11.0}
+GO_VERSION=${GO_VERSION:-1.17}
+GRPC_VERSION=${GRPC_VERSION:-1.42}
+GRPC_JAVA_VERSION=${GRPC_JAVA_VERSION:-1.42}
+GRPC_WEB_VERSION=${GRPC_WEB_VERSION:-1.3.0}
+GRPC_GATEWAY_VERSION=${GRPC_GATEWAY_VERSION:-2.7.0}
+UBER_PROTOTOOL_VERSION=${UBER_PROTOTOOL_VERSION:-1.10.0}
+SCALA_PB_VERSION=${SCALA_PB_VERSION:-0.11.6}
```

Comment:
I think that for some of the unrelated changes to go 1.17  like uber,scala, web (unless they are somehow related and I'm missing something), we should upgrade separately.

Reply:
Hmm, I just now noticed "web" in this comment; were you referring to GRPC_WEB_VERSION, changed in an earlier comment's chunk? I didn't change that back. Should I?

Reply:
(I'm testing that back-out locally, btw.)

Reply:
(It passed, so I'll go ahead and push that change, as I'm now convinced that's what you were asking me to do.)